### DISCUSSION TITLE
Any way to make this pop up look better?

### DISCUSSION THREAD

**Discussion by sethen** [company-mode/company-mode#1420] [General]
I love the documentation that is provided when I am on a candidate for completion but the pop up doesn't look great... is there any way to make it look better?

![Screenshot from 2023-10-27 01-05-10](https://github.com/company-mode/company-mode/assets/1493530/b33bd206-2940-4b2c-a44c-044c48f210cf)

**Comment by dgutov** [comment]
Hi! What kind of decorations do you have in mind?

If you're talking about the first line, it's printed by `company-posframe` (external package) and can be hidden by customizing `company-posframe-quickhelp-show-header`.
